index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+
+	<title>My Portifolio Page</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+	<div class="main">
+		<section class="toolbar">
+		<div class="logo">
+			<h1>My Logo</h1>
+		</div>
```

Comment:
Let's wrap the logo in an `a` tag, so when the visitor clicks on it, he/she is taken back to the top/home page.